template/build/webpack.base.conf.js

```diff
@@ -58,7 +58,7 @@ module.exports = {
           postcss: cssUtils.postcss,
           loaders: merge({js: 'babel-loader'}, cssUtils.styleLoaders({
             sourceMap: useCssSourceMap,
-            extract: env.prod
+            extractCSS: env.prod
```

Comment:
You are totally right. I misread how vue-loader was used. The problem I am trying to solve is that my component styles get added to my final css before quasar styles, meaning quasar styles take precedence over my own styles. Consider the following example:

```
<div class=“card my-card-style”>
    …
</div>

.my-card-style {
    margin-bottom: 50px;
}
```

The `.card` selector will take precedence over my own `.my-card-style`, making my custom rules overwritten by the framework.

What I see I did here is make the extract option of `csssUtils.styleLoaders` be always false, using always 'vue-style-loader' and having my vue components include their css in the JS, not in the global css file I'm generating, so the precedence was right for me.

I'll try to fix it some other way, you can close this PR. Sorry!